frontend/public/components/operator-lifecycle-manager/_operator-lifecycle-manager.scss

```diff
@@ -67,85 +42,52 @@ $clusterserviceversion-list-border: #ccc;
   font-size: 13px;
 }
 
-.co-clusterserviceversion-list-item__actions {
-  align-items: center;
+.co-crd-card-row {
   display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 15px;
+}
+
+.co-crd-card {
```

Comment:
> We should consider using the same cards as marketplace / catalog for consistency

(I don't know how hard this would be and wouldn't block the PR on it.)

Reply:
I'm open to addressing this later and not blocking the PR on design.